src/main/resources/translator.properties

```diff
@@ -890,8 +885,16 @@ au.com.rayh.XCodeBuilder.interpretTargetAsRegEx = interpretTargetAsRegEx
 
 au.com.rayh.XCodeBuilder.ipaManifestPlistUrl = ipaManifestPlistUrl
 
+jenkins.triggers.ReverseBuildTrigger = upstream
+#jenkins.triggers.ReverseBuildTrigger.type = OBJECT
```

Comment:
Do we want to leave this commented out?

Reply:
Oh, I didn't catch this. I just removed that line. This is formatted to match the [built-in DSL](https://jenkins.tinyspeck.com/plugin/job-dsl/api-viewer/index.html#method/javaposse.jobdsl.dsl.helpers.triggers.TriggerContext.upstream) which is not an object.